buildSrc/src/main/java/com/amazonaws/blox/swagger/ApiGatewayExtensionsFilter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.swagger;
+
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Swagger;
+import java.util.HashMap;
+import java.util.Map;
+import org.gradle.api.tasks.Input;
+
+/**
+ * Add API Gateway extensions to all operations of a swagger spec.
+ *
+ * <p>This will add the necessary x-amazon-apigateway-integration section to all operations in a
+ * Swagger spec to correctly proxy all requests to a single lambda function.
+ */
+public class ApiGatewayExtensionsFilter implements SwaggerFilter {
```

Comment:
I was mostly concerned about code size because we have to keep the jar as small as possible for Lamdba. What do you think? I think a separate Maven repo is overkill, however.